core/forms.py

```diff
@@ -14,6 +18,78 @@ def clean(self, values):
         return super(BetterReCaptchaField, self).clean(values)
 
 
+class AddOrganizerForm(forms.Form):
+    """
+    Custom form for adding new organizers to an existing event.
+
+    If user of given email already exists, they're added to the event and
+    receive e-mail notification about it.
+
+    If user is new, they're created (randomly generated password), invited
+    to Slack and receive e-mail notification with instructions to login
+    (including password).
+    """
+    event = forms.ModelChoiceField(queryset=Event.objects.all())
+    name = forms.CharField(label="Organizer's full name")
```

Comment:
I would change it for "Organizer's first and last name". That way, we will have the full name (some organizers only send the first name when I ask for full name) and it will be in the order we need.